bundles/core/org.eclipse.smarthome.core.thing.test/src/test/java/org/eclipse/smarthome/core/thing/firmware/FirmwareUpdateServiceTest.java

```diff
@@ -149,13 +149,14 @@ public void setup() {
 
         firmwareRegistry = new FirmwareRegistry();
         firmwareUpdateService.setFirmwareRegistry(firmwareRegistry);
+        firmwareUpdateService.setLocaleProvider(mockLocaleProvider);
         firmwareRegistry.setLocaleProvider(mockLocaleProvider);
 
-        when(mockProvider.getFirmware(eq(FW009_EN.getUID()), Matchers.any(Locale.class))).thenReturn(FW009_EN);
-        when(mockProvider.getFirmware(eq(FW111_EN.getUID()), Matchers.any(Locale.class))).thenReturn(FW111_EN);
-        when(mockProvider.getFirmware(eq(FW112_EN.getUID()), Matchers.any(Locale.class))).thenReturn(FW112_EN);
-        when(mockProvider.getFirmware(eq(FWALPHA_EN.getUID()), Matchers.any(Locale.class))).thenReturn(FWALPHA_EN);
-        when(mockProvider.getFirmwares(any(ThingTypeUID.class), any(Locale.class))).then(invocation -> {
+        when(mockProvider.getFirmware(eq(FW009_EN.getUID()), ArgumentMatchers.any())).thenReturn(FW009_EN);
+        when(mockProvider.getFirmware(eq(FW111_EN.getUID()), ArgumentMatchers.any())).thenReturn(FW111_EN);
+        when(mockProvider.getFirmware(eq(FW112_EN.getUID()), ArgumentMatchers.any())).thenReturn(FW112_EN);
+        when(mockProvider.getFirmware(eq(FWALPHA_EN.getUID()), ArgumentMatchers.any())).thenReturn(FWALPHA_EN);
+        when(mockProvider.getFirmwares(any(ThingTypeUID.class), any())).then(invocation -> {
```

Comment:
Here we have a strange mix of ArgumentMatchers.any() and org.hamcrest.CoreMatchers.any(). Can this be reduced to one (preferably static) import?

Reply:
Probably yes. Let me check what the IDE did there and why it didn't do so consistently.

Reply:
Some more insights on that: `any()` _always_  comes from `ArgumentMatchers` and indeed, could be used as a static import. Hamcrest only has `any(Class)` - which Mockito also has and there static imports cannot be used (at least not if both are statically imported) because they're ambiguous. In any case, Hamcrest and Mockito matchers are not interchangeable.

I'll clean that up in the follow-up PR though.

Reply:
PS: Correction - it works with `any(Class)` because they have different return types, but doesn't work with `isA(Class)`.